src/Aspire.Hosting.Azure.CognitiveServices/PublicAPI.Shipped.txt

```diff
@@ -14,4 +14,3 @@ Aspire.Hosting.ApplicationModel.AzureOpenAIResource.Deployments.get -> System.Co
 Aspire.Hosting.AzureOpenAIExtensions
 static Aspire.Hosting.AzureOpenAIExtensions.AddAzureOpenAI(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.AzureOpenAIResource!>!
 static Aspire.Hosting.AzureOpenAIExtensions.AddAzureOpenAI(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name, System.Action<Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.AzureOpenAIResource!>!, Aspire.Hosting.ResourceModuleConstruct!, Azure.Provisioning.CognitiveServices.CognitiveServicesAccount!, System.Collections.Generic.IEnumerable<Azure.Provisioning.CognitiveServices.CognitiveServicesAccountDeployment!>!>? configureResource) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.AzureOpenAIResource!>!
-static Aspire.Hosting.AzureOpenAIExtensions.AddDeployment(this Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.AzureOpenAIResource!>! builder, Aspire.Hosting.ApplicationModel.AzureOpenAIDeployment! deployment) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.AzureOpenAIResource!>!
```

Comment:
Are we able to do this without making breaking changes? For example, can we obsolete something and add new APIs?

Reply:
I reverted as I didn't remember why the change was done. Do we want to keep `AddDeployment` or replaced it with `WithDeployment`?